code/src/java/pcgen/io/ExportHandler.java

```diff
@@ -336,8 +336,10 @@ private void exportCharacterUsingFreemarker(PlayerCharacter aPC, BufferedWriter
 
 			// data-model
 			Map<String, Object> pc = OutputDB.buildDataModel(aPC.getCharID());
+			Map<String, Object> mode = OutputDB.buildModeDataModel();
```

Comment:
Would it be better to have this call pass in the game mode to be used? That way the assumption of SettingsHandler.getGame() is localised.